pycodestyle.py

```diff
@@ -1476,10 +1485,15 @@ def _is_eol_token(token, _eol_token=_is_eol_token):
 
 def _get_parameters(function):
     if sys.version_info >= (3, 3):
-        return [parameter.name
-                for parameter
-                in inspect.signature(function).parameters.values()
-                if parameter.kind == parameter.POSITIONAL_OR_KEYWORD]
+        try:
+            return [parameter.name
+                    for parameter
+                    in inspect.signature(function).parameters.values()
+                    if parameter.kind == parameter.POSITIONAL_OR_KEYWORD]
+        except ValueError:
+            """Catches "No signature found for builtin <built-in method
```

Comment:
It's in fact no longer needed since the merge (kina makes me happy ^^). Rebased on master.